omnibus/package-scripts/agent/preinst

```diff
@@ -45,6 +45,12 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         fi
     fi
 
+    PIP_PATH=$INSTALL_DIR/embedded/bin/pip
+    if [ -x $PIP_PATH ]; then
+        echo "Resetting integrations..."
+        $PIP_PATH freeze | grep datadog | grep  -v datadog-checks-base | xargs $PIP_PATH uninstall -y -q --no-cache-dir
```

Comment:
nits:

* the 1st `grep` could be a bit more restrictive (`'^datadog-'` maybe?)
* 2 spaces between `grep` and `-v`  😛 
* on the pip command: I'd add `|| true` to it to make sure that the install does not fail because for some reason `pip` failed to uninstall an integration.

Reply:
done